### ISSUE TITLE
OpenJDK 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0

### ISSUE THREAD

**Issue by mathieuchateau** [rundeck/rundeck#1367]
Hello,

I have this warning when I start any cli command:

OpenJDK 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0

Can I safely remove it from profile file in rundeck?

**Comment by ahonor** [comment]
Yes, you can. We will need to remove it from the template, too. 


     On Friday, July 31, 2015 6:47 AM, Mathieu Chateau <notifications@github.com> wrote:
   

 Hello,I have this warning when I start any cli command:OpenJDK 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0Can I safely remove it from profile file in rundeck?—

**Comment by gschueler** [comment]
@mathieuchateau  we are requiring java 8 in 2.8 release, so I will reopen this to track adding this change to the profile for that release

**Comment by senselesss64** [comment]
yes
